package/fedora/freecad.spec

```diff
@@ -275,11 +315,21 @@ popd
 # Remove obsolete Start_Page.html
 rm -f %{buildroot}%{_docdir}/%{name}/Start_Page.html
 # Belongs in %%license not %%doc
+# Well since it is trying to open it from "Help > About FreeCAD > Libraries", 
+# and there is only 3D Mouse Support under "Help > About FreeCAD > License", 
+# it is better to keep it as documentation for now /ljo
+%if 0%{?fedora} < 39
 rm -f %{buildroot}%{_docdir}/freecad/ThirdPartyLibraries.html
+%endif
```

Comment:
I'm confused as to why this was ever removed in the first place, IMO this should always be kept

Reply:
Thanks, yes, it felt strange, so I brought it back. As long as the tab is there it feels strange to have it empty and failing retrieval. But since it was tested with 38 previously I added the version check to get it consistent with how that previous version behaved. I will align with the F40+ changes above, though.

Reply:
Fixed